app/controllers/public.js

```diff
@@ -1,24 +1,31 @@
+import classic from 'ember-classic-decorator';
+import { action, computed } from '@ember/object';
 import Controller from '@ember/controller';
 import moment from 'moment';
-import { computed } from '@ember/object';
 
-export default Controller.extend({
-  smallLead: computed('session.currentRouteName', function() {
+@classic
+export default class PublicController extends Controller {
+  @computed('session.currentRouteName')
+  get smallLead() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected getter 'smallLead' to always return a value.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5274346)